docs/source/teams/installation.rst

```diff
@@ -223,6 +223,32 @@ You can do this in any of the following ways:
 `boto3 <https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#configuring-credentials>`_
 python library.
 
+For convenience, below are the environment variables that can be set to
+use some of the AWS-supported credentialing methods. This list is subject to
+change; please see the `boto3` link above for the up-to-date reference.
+
+.. code-block:: bash
+
+    # Access Key
+    export AWS_ACCESS_KEY_ID=...
+    export AWS_SECRET_ACCESS_KEY=...
+    export AWS_SESSION_TOKEN=... # if applicable
+    export AWS_DEFAULT_REGION=...
+
+    # Web Identity Provider
+    export AWS_ROLE_ARN=...
+    export AWS_WEB_IDENTITY_TOKEN_FILE=...
+    export AWS_ROLE_SESSION_NAME... #if applicable
+    export AWS_DEFAULT_REGION=...
+
+    # Shared Credentials File
+    export AWS_SHARED_CREDENTIALS_FILE="/path/to/aws-credentials.ini"
+    export AWS_PROFILE=default  # optional
+
+    # AWS Config File
```

Comment:
This is now listed twice. Looks like in the Azure section we split raw creds and `.ini` files into separate numbered sections. Do the same here for consistency?